<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1033</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>課程管理系統</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>171</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="btnShow">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>111</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>列出所有課程</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnCreate">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>新增課程</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDelete">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>刪除課程</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>160</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>編輯課程</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="tableClass">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>821</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1033</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
